src/ray/core_worker/reference_count.cc

```diff
@@ -1151,9 +1151,13 @@ Status ReferenceCounter::FillObjectInformation(
   absl::MutexLock lock(&mutex_);
   auto it = object_id_refs_.find(object_id);
   if (it == object_id_refs_.end()) {
-    return Status::ObjectNotFound("Object " + object_id.Hex() + " not found");
+    RAY_LOG(WARNING) << "Object locations requested for " << object_id
```

Comment:
Same comment as below, but I've seen this happens a lot I think because this is a common scenario; 

1. OBOD send a request
2. Object evicted
3. OBOD request arrived
4. Then it is no-op

I think we are handling this in the low level layer (object manager)? so my impression is having WARNING logs can have misleading information

Reply:
Actually I don't understand why this is happening a lot? We shouldn't be evicting the object until other refs are out of scope, so in the common case, OBOD requests should always succeed.

Reply:
Yeah this isn't a WARN, it's an expected race condition (DEBUG).

The reason it can always happen is if OBOD RPCs arrive late, or stale object fetches in pull manager. In general, we should avoid these kind of spammy warning messages when the system can handle stale/outdated requests, it's bitten us several times in the past.

Reply:
Got it, thanks!